controllers/permits/special-uses/non-commercial/index.js

```diff
@@ -43,8 +44,85 @@ get.all = function(req,res){
 // get id
 
 get.id = function(req,res){
+	
+	var jsonData = {};
+
+	var jsonResponse = {};
+	jsonResponse['success'] = false;
+	jsonResponse['api'] = 'FS ePermit API';
+	jsonResponse['type'] = 'controller';
+	jsonResponse['verb'] = 'get';
+	jsonResponse['src'] = 'json';
+	jsonResponse['route'] = 'permits/special-uses/commercial/outfitters/{controlNumber}';
+	
+	jsonData.response = jsonResponse;
+
+	var cnData = noncommercial_data[1095010356];
+
+	console.log('cnData='+cnData);
+
+	if(cnData){
+
+		console.log('data of accinstCn='+cnData.accinstCn);
+
+		jsonResponse['success'] = true;
```

Comment:
Looks good. Just watch it when using tabs vs spaces. Github shows tabs a bit weird, so you get the formatting looking off like this when it looks line in a text editor.